ReSwiftRouterTests/ReSwiftRouterTestsUnitTests.swift

```diff
@@ -119,6 +119,42 @@ class ReSwiftRouterUnitTests: QuickSpec {
                 expect(action1Correct).to(beTrue())
                 expect(action2Correct).to(beTrue())
             }
+            
+            it("generates a Change action on the last common subroute, also for routes of different length, old route is longer than the new one") {
```

Comment:
This is quite a mouthful :) I suggest to rename the three cases to:

```swift
    it("generates a Change on the last common subroute for routes of same length") { ... }
    it("generates a Change on the last common subroute when new route is longer than the old route") { .. }
    it("generates a Change on the last common subroute when the new route is shorter than the old route") { ... }
```

Maybe you can come up with a shared context that fits, like `context("changing non-root elements")` or so.

Reply:
@DivineDominion - changed the test names as you have suggested.